manifests/plugins.pp

```diff
@@ -28,6 +28,8 @@
 # @param dependencies
 #   Package dependencies needed to install plugins and extensions.
 #   Default is OS dependent.
+# @param gem_dependencies
```

Comment:
This should be added to the README since it is a common use case.

Reply:
Added README example of usage